test/unit/uk/gov/hmrc/apidocumentation/models/DocumentationSpec.scala

```diff
@@ -295,6 +295,41 @@ class DocumentationSpec extends UnitSpec {
 
       result shouldBe expected
     }
+
+    "include REST and Test Support APIS" in {
+      val restApi = APIDefinition("serviceName", "restApi", "description", "context1", None, None, Seq(APIVersion("1.0", None, STABLE, Seq.empty)))
+      val testSupportApi = APIDefinition("serviceName", "testSupportApi", "description", "context2", None, isTestSupport = Some(true), Seq(APIVersion("1.0", None, STABLE, Seq.empty)))
+      val categoryMap = Map(
```

Comment:
I think it would be good to make a function that generates an api / restApi / testSupportApi